packages/dart/noports_core/lib/src/srv/srv_impl.dart

```diff
@@ -630,69 +637,111 @@ class SrvImplDart implements Srv<SocketConnector> {
     return socketConnector;
   }
 
-  Future _handleMultiConnectRequest(
+  Future<void> _handleMultiConnectRequest(
```

Comment:
refactor - `_handleMultiConnectRequest` is now just handling a multi connect request; the parsing of what was received on the control socket is now done in the new `_handleControlRequest` function